src/benchmarks/micro/corefx/System.Collections/Perf.BitArray.cs

```diff
@@ -14,7 +14,7 @@ public class Perf_BitArray
         private const int DefaultShiftCount = 17;
         private const bool BooleanValue = true;
 
-        [Params(Utils.DefaultCollectionSize)]
+        [Params(4, Utils.DefaultCollectionSize)]
```

Comment:
could you please add a comment explaining why we have two test cases? Something similar to:

https://github.com/dotnet/performance/blob/cc73a0126ae9056382fe23a9dee183b18b1b67de/src/benchmarks/micro/corefx/System.Globalization/StringHash.cs#L33-L36